charts/vela-core/templates/defwithtemplate/notification.yaml

```diff
@@ -56,44 +56,41 @@ spec:
         		// +usage=Specify the message that you want to sent, refer to [dingtalk messaging](https://developers.dingtalk.com/document/robots/custom-robot-access/title-72m-8ag-pqw)
         		message: {
         			// +usage=Specify the message content of dingtalk notification
-        			text?: *null | close({
+        			text?: close({
         				content: string
         			})
         			// +usage=msgType can be text, link, mardown, actionCard, feedCard
         			msgtype: *"text" | "link" | "markdown" | "actionCard" | "feedCard"
-        			link?:   *null | close({
+        			#link:   close({
```

Comment:
`#` means a close struct, there's no need to use `close` here.

Reply:
Oops.. Thanks!